lib/classes/Variables.js

```diff
@@ -170,6 +173,23 @@ class Variables {
     return valueToPopulate;
   }
 
+  getValueFromImports(variableString) {
```

Comment:
This method checks if the user who want's to use `imports` via the variable system has explicitly defined what he want's to use in the `service.imports` config.